lib/utils.js

```diff
@@ -64,13 +64,32 @@ function cloneContext(context, params = {}) {
 }
 
 /**
+ * Strips env values from the options object
+ *
+ * @param {object} options Optional options passed to spawn
+ */
+function convertOptionsForError(options = {}) {
+  const override = options.env && { env: Object.keys(options.env) };
```

Comment:
It will break if you pass `null` as `options`, but it's up to you if you want to cover this edge-case

Reply:
good eye, but let's take that risk for now :)